x-pack/plugins/security_solution/public/management/hooks/use_with_show_responder.tsx

```diff
@@ -81,18 +82,21 @@ export const useWithShowResponder = (): ShowResponseActionsConsole => {
           'data-test-subj': `${agentType}ResponseActionsConsole`,
           storagePrefix: 'xpack.securitySolution.Responder',
           TitleComponent: () => {
-            if (agentType === 'endpoint') {
-              return <HeaderEndpointInfo endpointId={agentId} />;
-            }
-            if (agentType === 'sentinel_one') {
+            if (agentStatusClientEnabled || agentType === 'sentinel_one') {
               return (
-                <HeaderSentinelOneInfo
+                <AgentInfo
                   agentId={agentId}
+                  agentType={agentType}
                   hostName={hostName}
                   platform={props.platform}
```

Comment:
Since you destructured platform from props, it can be used here :)

Reply:
oh yeah! I missed that. Thanks for noticing.

Reply:
fixed